### ISSUE TITLE
Keep track of needed wildcards

### ISSUE THREAD

**Issue by tooomm** [Senryoku/MTGADraft#212]
With the collection status showing how many copies of a card you already own in every booster, I came up with the idea to keep track of the wildcards needed to craft missing cards and display that information somewhere during draft/deck building.

**Comment by Senryoku** [comment]
![image](https://user-images.githubusercontent.com/1338143/107550941-f3f21e80-6bd1-11eb-8ba1-14a928ed9a83.png)
?

**Comment by tooomm** [comment]
👍 Left feedback in the PR.

Interesting question:
What happens if you own a particular card 3x already and end up drafting it 5 times in a draft on your site...
Do you only need to spend 1 wildcard to get the maximum amount of copies and the rest is free in MTGA?

**Comment by Senryoku** [comment]
> 👍 Left feedback in the PR.

Thanks!

> Interesting question:
> What happens if you own a particular card 3x already and end up drafting it 5 times in a draft on your site...
> Do you only need to spend 1 wildcard to get the maximum amount of copies and the rest is free in MTGA?

I think once you own 4 copies, you effectively own an unlimited amount (see Persistent Petitioners), will update to reflect that too :)

**Comment by tooomm** [comment]
First quick feedback after the merge:

- I like that the new wildcard warning during the draft is all a single tooltip - it was easy to miss with the two right next to each other before 👌 
- Normal basic lands are falsely counted towards common wildcards, same happens for snow basics which doesn't require them either
- The overview of needed rare/mythic wildcards in the menu headline doesn't account for cards in the sideboard, same goes for the color change in general:
![sb](https://user-images.githubusercontent.com/9874850/107880552-d24dab80-6edf-11eb-8f88-a36340c8e463.png)
Intentional maybe?
- Patch notes could be confusing now as the last one supersedes the former note:
```
14/02/2021
    Implemented wildcard tracking while drafting and brewing.

30/01/2021
    Added a reminder of the number of owned copies of each card while drafting.
```

**Comment by Senryoku** [comment]
> * Normal basic lands are falsely counted towards common wildcards, same happens for snow basics which doesn't require them either

Fixed in 30b2638e57be64ac3845273a556a1a7b7535ad82

> * The overview of needed rare/mythic wildcards in the menu headline doesn't account for cards in the sideboard, same goes for the color change in general:
> Intentional maybe?

Yep, see reasoning in PR
